bgpd/bgp_bmp.c

```diff
@@ -2553,6 +2806,70 @@ static int bgp_bmp_init(struct event_loop *tm)
 	return 0;
 }
 
+/* TODO remove "bn", redundant with updated_route->net ? */
```

Comment:
forgot to leave that out. i meant remove from the function parameters, not from route table. i like to keep it just in case one day old_select and new_select end up being both NULL